src/OpenLoco/GameCommands/Cheat.cpp

```diff
@@ -51,7 +51,17 @@ namespace OpenLoco::GameCommands
                 if (vehicle->owner != targetCompanyId)
                     continue;
 
-                vehicle->owner = ourCompanyId;
+                Vehicles::Vehicle train(vehicle);
+                train.head->owner = ourCompanyId;
+                train.veh1->owner = ourCompanyId;
+                train.veh2->owner = ourCompanyId;
+                train.tail->owner = ourCompanyId;
+
+                for (auto& car : train.cars)
```

Comment:
This isn't setting them all. You need to iterate the car components as well

Reply:
Oh, I thought this would already iterate all car components. How should I approach it instead?

Reply:
you just need to do:

```
for (auto& car : train.cars)
{
    for(auto& carComp : car)
    {
        carComp.front->owner = ...
        carComp.back->owner = ...
        carComp.body->owner = ...
    }
}

Reply:
Thanks, that fixes the vehicles crashing.